tests/unittests/paybuttonsService.test.ts

```diff
@@ -1,82 +1,34 @@
-import models from 'db/models/index'
+import prisma from 'prisma/clientInstance'
 import * as paybuttonsService from 'services/paybuttonsService'
-import * as dbMocks from 'tests/dbMocks'
-
-const mockModels: any = models
-mockModels.paybuttons.findOne = jest.fn(dbMocks.mockedPaybuttonsFindOne)
-mockModels.paybuttons.findAll = jest.fn(dbMocks.mockedPaybuttonsFindAll)
-mockModels.sequelize.transaction = jest.fn((_) => { return { id: 4 } })
+import { prismaMock } from 'prisma/mockedClient'
+import { mockedPaybutton, mockedPaybuttonList, mockedChain } from '../mockedModels'
 
 describe('Fetch services', () => {
   it('Should fetch paybutton by id', async () => {
-    const result = await paybuttonsService.fetchPaybuttonById('4')
-    expect(result).toEqual(expect.objectContaining({
-      id: 4,
-      providerUserId: expect.any(String),
-      createdAt: expect.any(String),
-      updatedAt: expect.any(String)
-    }))
-  })
+    prismaMock.paybutton.findUnique.mockResolvedValue(mockedPaybutton)
+    prisma.paybutton.findUnique = prismaMock.paybutton.findUnique
```

Comment:
There are a few tests here which don't expect anything

Reply:
There is a `expect(result).toEqual(mockedPaybutton)` below